store/README.md

```diff
@@ -0,0 +1,167 @@
+# Store
+
+**Store** maintains the state of the chain. It serves as the "source of truth" for the chain - i.e., if it is not in 
+the store, the node does not consider it to be a part of the chain. 
+**Store** is one of components of the [Miden node](..).
+
+## Architecture
+
+The Miden node is still under heavy development and current architecture is subject of change. This topic will be 
+filled later.
+
+## Usage
```

Comment:
If the section is Usage, then I recommend to add 2 sub-sections:

Installing the Store
Running the Store